techmatters/flex-plugins
# CHI-1785: Fix emojis for AseloMessageInput

Review after #1335 is merged

## Description

* EmojiPicker now part of AseloMessageInput
* Updates the redux state for the draft message when part of an Aselo messaging UI
* Draft text now updated in redux on blurring the text area, not unloading the component, to ensure the emoji picker always has up to date content to append an emoji to.
* EmojiPicker will not put a space between the content & the emoji if the content already ends in a space
* Canned responses & EmojiPicker now respect feature flag in AseloMessageInput

### Checklist
- N/A New tests added
- N/A Strings are localized
- N/A Tested for call contacts

### Related Issues
CHI-1785

### Verification steps

* Verify emoji picker works as expected with the Aselo Messaging UI
* Verify emoji picker still works as expected with the default Twilio messaging UI
* Verify that the various combinations of feature flags work as expected (i.e. Canned Responses & Emoji Picker enable & disable as expected in both messaging UIs)
